mne_bids/read.py

```diff
@@ -331,6 +331,8 @@ def _handle_scans_reading(scans_fname, raw, bids_path):
         if "." not in acq_time:
             # acquisition time ends with '.%fZ' microseconds string
             acq_time += ".0Z"
+        if "Z" not in acq_time:
```

Comment:
I don't think we should do this. If the Z is not there, adding it will alter the time

and we don't want to cause disruptions to the time-space continuum, do we? 😳

Reply:
Z indicates a time zone, it has nothing to do with the nanosecond (?) bit at the end of the time string per se

Reply:
> this also means that the line above is actually buggy too

you are absolutely right 😳  I simply followed the existing logic without questioning it.

Reply:
yes no worries, these datetime strings are anything but intuitive! I also had to google a bit first…

Reply:
Question is, *must* we have datetime in UTC in the end? What does MNE expect?

Reply:
I won't have time to think about this in the next days/weeks, but I have it on my to do list.